container/karaf/src/main/filtered-resources/etc/org.opennms.features.geocoder.nominatim.cfg

```diff
@@ -1,2 +1,3 @@
 emailAddress=
 referer=
+useSystemProxy=false
```

Comment:
Can we add the default in the blueprint.xml instead of editing this configuration file?